aten/src/ATen/cpu/vec256/vec256_base.h

```diff
@@ -607,7 +606,20 @@ template <class T,
 Vec256<T> inline clamp(const Vec256<T> &a, const Vec256<T> &min_vec, const Vec256<T> &max_vec) {
```

Comment:
This looks great, but is this code path actually exercised if complex inputs to clamp are disallowed?

Reply:
I am not 100% sure. It seems clamp kernels are sometimes used in other operations, e.g. ReLU. I am not sure if there is any kernel that would require complex inputs. I can try removing this and see if there are any failing tests?

Reply:
Yeah, please try removing it so we don't cause confusion.

Reply:
Ok, removed all implementations of complex clamp.